libs/device/feature/src/lib/restore/restore.service.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Result, ResultObject } from "Core/core/builder"
+import { IpcEvent } from "Core/core/decorators"
+import { AppError } from "Core/core/errors"
+import { DeviceManager } from "Core/device-manager/services"
+import { DeviceId } from "Core/device/constants/device-id"
+import { ApiResponse } from "Core/device/types/mudita-os"
+import {
+  APIRestoreServiceEvents,
+  GeneralError,
+  PreRestore,
+  PreRestoreValidator,
+  Restore,
+  RestoreValidator200,
+  RestoreValidator202,
+} from "device/models"
+import { random } from "lodash"
```

Comment:
🟡 
Using `import { random } from "lodash";` potentially imports more code than necessary. For better bundle optimization, recommend importing only the required function directly: `import random from 'lodash/random';`. This will reduce the final bundle size.